src/pages/project-list/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { basicWrap } from 'styles/containers';
+import styled from '@emotion/styled';
+import { Heading4 } from 'styles/typography';
+import Empty from './empty';
+import Card from './card';
+
+const DUMMY = [
+  {
+    id: 1,
+    name: '강원도 우정여행',
+    startDate: '2021-05-13',
+    endDate: '2021-05-13',
+    memberCount: 1,
+    give: 20000,
+    take: 400000
+  }
+];
+
+const DUMMY2 = [];
+
+const Title = styled(Heading4)`
+  margin: 16px 0;
+`;
+
+export default function ProjectList() {
+  if (DUMMY2.length) {
+    return <Empty />;
+  }
+  return (
+    <div css={basicWrap}>
+      <Title>여행 중</Title>
+      {DUMMY.length && DUMMY.map
```

Comment:
```suggestion
      {DUMMY.length === 0 && DUMMY.map
```

마이너) 요렇게 명확한 비교를 해주는 것은 어떨까유? 저 표현은 `DUMMY.length` 가 nullish 한 상황도 함께 표현하는 것이라 `DUMMY가 없는 빈 배열`을 표현하려면 길이를 명확히 비교해주는 것도 좋은 것 같아요!